index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+   <header class="header-container">
+    <nav class="nav">
+        <a>My logo</a>
+        <img src="/images/hamburger.png" alt="">
+    </nav>
+    <section class="headline">
+        <a  class="greetings">I’m Mariane Glad to see you!</a>
+        <p class="welcome-message">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <h1 class="heading-1">LET’S CONNECT</h1>
```

Comment:
As the previous reviewer suggested, please change this from a `<h1>` tag to an `<a>` tag or any other heading tag🙏. Its considered best practice for a webpage to have only one `<h1>` tag, which describes the main content of the page. Also, it improves the page's SEO